boa3/model/builtin/native/nativecontract.py

```diff
@@ -10,11 +11,13 @@ class NativeContract:
 
     # Interop Types
     BlockType = BlockType.build()
+    RoleType = RoleType()
     TransactionType = TransactionType.build()
```

Comment:
I ended letting this pass in the other code review by mistake
Here it's better to use the objects in `Interop` and `Builtin` classes to ensure that the same default type is used during type checking analysis